src/viz/viewer_system.cpp

```diff
@@ -42,6 +43,11 @@ inline void instanceTransformSetup(Context &ctx,
     };
 }
 
+uint32_t *VizRenderingSystem::getVoxelPtr(Context &ctx) {
```

Comment:
I believe the style in most of the code base on return types that are pointers is to have a space after the * (as well as before).